### DISCUSSION TITLE
Unable to assign Auth0 user groups

### DISCUSSION THREAD

**Discussion by P1xelC0da** [requarks/wiki#6508] [Help / Questions] [answered]
I added a Auth0 authentication source, and added the user, but I cannot seem to assign groups. Edit: It says that I can't modify it from that panel, where would I manage it at?

![image](https://github.com/requarks/wiki/assets/123631024/e5e09e1a-0184-4b14-9b83-5b4bd3d708e5)

**Comment by P1xelC0da** [comment]
uploaded the wrong image 💀

**Comment by NGPixel** [comment, marked as answer]
Users can only be added to the Administrators from Groups > Administrators > Users.

**Comment by P1xelC0da** [comment]
Thank you!

---
Damian Freeman
Executive Director of Engineering, Chair of the Board of Directors
Devsaurus
